### DISCUSSION TITLE
Images are not Loading on my Website

### DISCUSSION THREAD

**Discussion by brxic** [community/community#113173] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hey Guys,

I wrote a Website in VS-Code in HTML CSS and JS. When i launch my Website from VS-Code all the Images are loading as well as the favicon. But as I wanted to publish my Website on Github, my Images aren't loading. 

Can someone help me with a solution ? 

https://github.com/brxic/brxic.github.io

**Comment by santoshvandari** [comment, marked as answer]
Mostly, The main reason is the Path Issue. I suggest you use the absolute path rather than the relative path.
And I have visited your website, it working fine in my system. May be an issue with your browser. YOu can try it by using a different browser.

**Comment by Zain-Muhammad** [comment]
Have you fixed it I hit your repo link but found 404.
If you have still facing this issue then resolve 404 and let me know I'll help you to fix it out.
![image](https://github.com/community/community/assets/68952585/c9e26edb-c169-47bb-b1ad-62ec3a4a1604)

**Reply by brxic** [reply]
sorry i deleted my repository because i gave up but now i created a new one and fixed the problem except for the favicon but thats not important. thx for your offer to help
